.github/workflows/release.yml

```diff
@@ -3,27 +3,28 @@ name: goreleaser
 on:
   push:
     tags:
-      - '*'
+      - 'v*.*.*'
 
 jobs:
   goreleaser:
     runs-on: ubuntu-latest
     steps:
-      -
-        name: Checkout
+      - name: Checkout
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      -
-        name: Set up Go
+      - name: Set up Go
         uses: actions/setup-go@v2
         with:
           go-version: 1.16
-      -
-        name: Run GoReleaser
+      - name: Get packr
+        run: go get -u github.com/gobuffalo/packr/v2/packr2
+      - name: Prepare
+        run: git reset --hard
+      - name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v2
         with:
           version: latest
           args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
now that the repo is public, is this still necessary?

Reply:
we need it for we can publish the binary into the releases page